tests/view/items/test_items.py

```diff
@@ -0,0 +1,71 @@
+""" Tests for the items.
+"""
+from http import HTTPStatus
+from datetime import datetime
+
+from tests.view.items.mock_items import ItemMock, ItemsMock
+from timeless.items.views import ItemListView
+
+
+def test_list(client):
+    """ Test list is okay """
+    items = [
+        {
+            "id": 1,
+            "name": "Pancakes",
+            "stock_date": datetime.utcnow,
+            "comment": "Really delicious pancakes made from flour and eggs",
+            "company_id": 1,
+            "created_on": datetime.utcnow,
+            "updated_on": datetime.utcnow,
+            "employee_id": 1
+        },
+        {
+            "id": 2,
+            "name": "Dental Floss",
+            "stock_date": datetime.utcnow,
+            "comment": "Dental floss from Montana salesmen",
+            "company_id": 1,
+            "created_on": datetime.utcnow,
+            "updated_on": datetime.utcnow,
+            "employee_id": 1
+        },
+        {
+            "id": 3,
+            "name": "Power Cable",
+            "stock_date": datetime.utcnow,
+            "comment": "Power cable made of leather and heavy metal",
+            "company_id": 1,
+            "created_on": datetime.utcnow,
+            "updated_on": datetime.utcnow,
+            "employee_id": 1
+        },
+        {
+            "id": 4,
+            "name": "Brick Block",
+            "stock_date": datetime.utcnow,
+            "comment": "Red block made from bricks, sometimes hide coins",
+            "company_id": 1,
+            "created_on": datetime.utcnow,
+            "updated_on": datetime.utcnow,
+            "employee_id": 1
+        }
+    ]
+
+    assert client.get("/items", data=items).status_code == HTTPStatus.OK
```

Comment:
@paulodamaso It will not work. You can pass items in this way. You should create items in database. Look at this example https://github.com/timelesslounge/timelessis/blob/master/tests/integration/it_employees_test.py#L61

Reply:
@vladarefiev I don't want to create items in database. I want to use a mock objetct, this is an unit test, not an integration test. That's what I am triyng to do all this time.

Reply:
let's @emilianodellacasa decide

Reply:
@vladarefiev I agree with @paulodamaso, if this is a unit test database shouldn't be involved

Reply:
@vladarefiev @paulodamaso To be totally honest, in views I prefer using factories even if this means creating data on the db